domain/src/main/java/org/oppia/android/domain/profile/ProfileManagementController.kt

```diff
@@ -566,7 +567,8 @@ class ProfileManagementController @Inject constructor(
         )
       val updatedProfile = profile.toBuilder().apply {
         learnerId = when {
-          !learnerStudyAnalytics.value -> "" // There should be no learner ID if no ongoing study.
+          !enableLearnerStudyAnalytics.value -> ""
+          // There should be no learner ID if no ongoing study.
```

Comment:
Should be above the line above since that’s what is being documented.

Reply:
Done